Automattic/simplenote-ios
# Dropping VSTheme: Mark IV

### Fix
- **Editor:** Metrics are now ObjC constants: maximum width / padding
- **Finally drops VSTheme** 🤟

We're just focusing on wiring Constants (where the old VSTheme lookup was). Most of this "Regular" "Device" checks will become 💯 irrelevant the second SplitViewController is implemented.

cc @eshurakov **Last one, promise!!**
Thank youuuu!!

---

### Test: iPad
1. Launch the app on iPad
2. Open any note

- [x] Verify that the Editor is constrained to 640pt
- [x] Verify that the app reacts well to Adaptivity (Sideover + 1/3 + 2/3)
- [x] Verify that whenever you're in Multitasking the horizontal (text) padding matches the AppStore build (20 points!)

### Test: iPhone
1. Launch the app on iPad
2. Open any note

- [x] Verify that the editor looks exactly like the AppStore build please!

### Release
These changes do not require release notes.
